packages/applications-service-api/src/repositories/project.backoffice.repository.js

```diff
@@ -11,12 +11,64 @@ const getByCaseReference = async (caseReference) => {
 	});
 };
 
-const getAllApplications = async () => {
-	return prismaClient.project.findMany({
+const getAllApplications = async (options = {}) => {
+	const { filters, searchTerm, orderBy, offset, size } = options;
+	const where = {};
+
+	if (filters?.region || filters?.stage || filters?.sector || searchTerm) {
+		where['AND'] = [];
+	}
+
+	if (searchTerm) {
+		const terms = options.searchTerm.split(' ');
+		where['AND'].push({
+			OR: [
+				{ projectName: { contains: searchTerm } },
+				...terms.map((term) => ({
+					OR: [
+						{ applicant: { organisationName: { contains: term } } },
+						{ applicant: { firstName: { contains: term } } },
+						{ applicant: { lastName: { contains: term } } }
+					]
+				}))
+			]
+		});
+	}
+	if (filters?.region) {
+		where['AND'].push({
+			OR: filters.region.map((region) => ({
+				regions: { contains: region }
+			}))
+		});
+	}
+	if (filters?.stage) {
+		where['AND'].push({
+			OR: filters.stage.map((stage) => ({
+				stage: stage
+			}))
+		});
+	}
+	if (filters?.sector) {
+		where['AND'].push({
+			OR: filters.sector.map((sector) => ({
+				sector: { contains: sector }
+			}))
+		});
+	}
+
+	const findOptions = {
+		where,
+		...(orderBy && { orderBy }),
+		...(offset !== undefined && { skip: offset }),
+		...(size !== undefined && { take: size }),
 		include: {
 			applicant: true
 		}
-	});
+	};
+
+	const applications = await prismaClient.project.findMany(findOptions);
+	const count = await prismaClient.project.count({ where });
```

Comment:
I think this may producing counts that include the search term (filtering).
I think the counts are total numbers irrespective of searchTerm.  Possibly need to pop the searchTerm off the 'where'.  I may be wrong though.